bazel/repositories.bzl

```diff
@@ -850,9 +850,16 @@ def _com_googlesource_chromium_zlib():
     )
 
 def _com_github_google_quiche():
-    external_genrule_repository(
+    external_http_archive(
         name = "com_github_google_quiche",
-        genrule_cmd_file = "@envoy//bazel/external:quiche.genrule_cmd",
+        # Rewrite third_party includes and #pragma clang.
```

Comment:
Out of curiosity, why is this here instead of in the genrule file?

Reply:
Heh, you must have miss my whining on Chat :) The genrule approach works by taking a list of input files and transforming them into a distinct of output files. The tip-of-tree genrule moves the input files into a quiche/ directory. But since we actually moved the files in the underlying quiche repo, envoy no longer needs to do this.As a result, genrule no longer works. :( The quiche.genrule_cmd file is hence deleted in this PR. Thankfully, this "patch_cmds" approach does seem to work.

Reply:
Ah right, thanks!